src/lib/containers/GenericCard.tsx

```diff
@@ -469,7 +468,19 @@ export default class GenericCard extends React.Component<
       genericCardSchemaDefined.description,
       facetAliases,
     )
-
+    
+    let ctaHref: string | undefined = undefined, ctaTarget: string | undefined = undefined
+    if (ctaLinkConfig) {
+      const ctaLinkValue: string = data[schema[ctaLinkConfig.link]]
+      const { href: newCtaHref, target: newCtaTarget } = this.getLinkParams(
+        ctaLinkValue,
+        undefined, //card link config
+        data,
+        schema,
+      )
+      ctaHref = newCtaHref
+      ctaTarget = newCtaTarget
```

Comment:
Is there a better way of doing this?

Reply:
I am wondering if you want to set `ctaLinkValue` outside the `if (ctaLinkConfig)` block because I am guessing whether you have the `ctaLinkConfig`, you still want to surfac the `ctaLinkValue` right? Also, what happen if `data[schema[ctaLinkConfig.link]]` is not defined?  What do you see there?

Reply:
Without a ctaLinkConfig I don't know where to look for the ctaLinkValue.
Not sure what happens when data[schema[ctaLinkConfig.link]] is undefined, I'll test and make sure that it doesn't render a link in that case!

Reply:
Changed to fallback to empty string (like so many other cases in this file), which results in the link not rendering.